### PR TITLE
Add 88.0.4324.150-1.1 for macOS x86-64 and arm64 (GitHub Actions)

### PR DESCRIPTION
**Pull request by kramred**

### CODE REVIEW

File: config/platforms/macos/88.0.4324.150-1.1.ini [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[_metadata]
```
Comment (Eloston) [inline config/platforms/macos/88.0.4324.150-1.1.ini:1]: I'm thinking: Should we move the existing x86-based releases to a new subfolder?

Comment (kramred) [reply config/platforms/macos/88.0.4324.150-1.1.ini:1]: Yes, probably. I was thinking the same but didn't know if we should wait till arm64 macs are in the majority (in Apple's lineup) – but maybe now is better, then it's done and we won't have to remember doing it in the future.

Can you move the existing releases or shall I modify this PR?

Comment (Eloston) [reply config/platforms/macos/88.0.4324.150-1.1.ini:1]: Alright, I've moved them to a new subfolder `x86_64`. Please update accordingly, thanks!

Comment (kramred) [reply config/platforms/macos/88.0.4324.150-1.1.ini]: I've updated the PR.

It seems that a few of the older releases might be missing causing the CI/cite_generation to fail (e.g. 60.0.3112.113-1_macos)

Comment (Eloston) [reply config/platforms/macos/88.0.4324.150-1.1.ini]: Resolving that. Thanks for the heads up.

**Review by Eloston — APPROVED** [review]
Thanks!

**Pull request merged**